### PR TITLE
Fix typos in README.md

### PR DESCRIPTION
**Pull request by urumo**

### CODE REVIEW

**Comment by code-maid** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/0xF6/mana_lang/blob/master/.github/CLA.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   ***I have read the CLA Document and I hereby sign the CLA***
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

File: README.md [hunk 1]
```diff
@@ -116,14 +116,14 @@ mkdir output
...
-See into `output` folder  
+The `output` folder should contain:
 - ishtar.exe - main ishtar file
 - ishtar.exp - export metadata for main module
 - ishtar.lib - dynamic library for main module
 - ishtar.pdb - debug symbols
 
 
-#### Compile manac
+#### Compile manяc
```
Comment (0xF6) [inline README.md:126]:
```suggestion
#### Compile manac
```

File: README.md [hunk 2]
```diff
@@ -224,42 +224,42 @@ To do so, there are a few things you need to know about the code:
...
   Why is it called mana?
 A:
-  I liked it so much 🗿🗿🗿
-  So, I tried to choose a memorable and simple name with which it would be easy 
-  to come up with a meaningful name for tools (like the 'rune' package manager).
-  The original name was Wave Lang, but I didn't like it, and it was chosen by random 🙂. 
+  I liked it very much 🗿🗿🗿
+  So, I tried to choose a memorable name that would be easy
+  simple for the tools (like the 'rune' package manager).
+  The original name was Wave Lang, but I didn't like it, and it was chosen by a chat in Telegram 🙂.
```
Comment (0xF6) [inline README.md:245]:
```suggestion
  The original name was Wave Lang, but I didn't like it, and it was chosen by random.
```

File: README.md [hunk 3]
```diff
@@ -224,42 +224,42 @@ To do so, there are a few things you need to know about the code:
...
+  So, now I'm developing using the C# runtime and the std, but in version 2.0 I'm planning to completely move away from runtime dependencies.
 
-Q: 
-  This language really support quantum computing? 
+Q:
+  This language really support quantum computing?
 A:
-  Not now, but in future i planning in first add support Microsoft Quantum Simulator, next - support Azure Qunatum or IBM quantum cloud.
-  And after appearance of stationary quantum extension card (like PCEx128 😃), i add support i'il supporting them too.
+  Not now, but in future I'm planning to add support for Microsoft Quantum Simulator, next - support for Azure Qunatum or IBM quantum cloud.
```
Comment (0xF6) [inline README.md:261]:
```suggestion
  Not now, but in future I'm planning to add supportMicrosoft Quantum Simulator, next - support Azure Qunatum or IBM quantum cloud.
```

**Review by 0xF6 — CHANGES_REQUESTED** [review]

**Comment by urumo** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by 0xF6 — APPROVED** [review]

**Pull request merged**